src/lib389/cli/openldap_to_ds

```diff
@@ -110,26 +119,29 @@ parser.add_argument('--confirm',
         help="Confirm that you want to apply these migration actions to the 389-ds instance. By default no actions are taken."
 )
 parser.add_argument('--ignore-overlay', nargs='*',
-        help="Ignore the following openldap overlays from having their configuration migrated to equivalent 389-ds plugins."
+        help="Ignore the following openldap overlays from having their configuration migrated to equivalent 389-ds plugins. Valid options are memberof, refint, unique.",
+        default=[]
 )
 parser.add_argument('--ignore-schema-oid', nargs='*',
-        help="Ignore the following openldap schema attribute or class OIDS from being migrated to 389-ds. This *may* create inconsistent schema which could cause the migration to fail. Use with caution."
+        help="Ignore the following openldap schema attribute or class OIDS from being migrated to 389-ds. This *may* create inconsistent schema which could cause the migration to fail. Use with caution.",
+        default=[]
 )
 parser.add_argument('--ignore-attribute', nargs='*',
-        help="Ignore the following attributes from entries that are loaded from the ldif. For example, you may not want to import userPassword hashes."
-)
-parser.add_argument('--no-overlays',
-        default=True, action='store_false',
-        help="Do not migrate any openldap overlays to equivalent 389-ds plugins."
-)
-parser.add_argument('--no-schema',
-        default=True, action='store_false',
-        help="Do not migrate the schema from openldap to 389-ds. This may cause the ldif import to fail."
-)
-parser.add_argument('--no-indexes',
-        default=True, action='store_false',
-        help="Do not create any indexes in 389-ds as defined in openldap slapd.d"
+        help="Ignore the following attributes from entries that are loaded from the ldif. For example, you may not want to import userPassword hashes.",
+        default=[]
 )
+# parser.add_argument('--no-overlays',
+#         default=True, action='store_false',
+#         help="Do not migrate any openldap overlays to equivalent 389-ds plugins."
+# )
+# parser.add_argument('--no-schema',
+#         default=True, action='store_false',
+#         help="Do not migrate the schema from openldap to 389-ds. This may cause the ldif import to fail."
+# )
+# parser.add_argument('--no-indexes',
+#         default=True, action='store_false',
+#         help="Do not create any indexes in 389-ds as defined in openldap slapd.d"
+# )
```

Comment:
Is there any particular reason for commenting out the code? I think if it's needed it can be retrieved from the older revision.
But maybe I miss something...

Reply:
All good, you're right I don't need to leave it there. I removed it in the final version :)